Formula/imagemagick.rb

```diff
@@ -126,7 +122,8 @@ def install
     args << "--with-quantum-depth=#{quantum_depth}" if quantum_depth
 
     # versioned stuff in main tree is pointless for us
-    inreplace "configure", "${PACKAGE_NAME}-${PACKAGE_VERSION}", "${PACKAGE_NAME}"
+    inreplace "configure",
```

Comment:
That seems problematic to me since it makes it easier not to notice that a non-master branch has to be used if it's spilling off to the right, but ho hum :)